bedrock/configuration-files.md

```diff
@@ -11,13 +11,17 @@ published: true
 ---
 The root `web/wp-config.php` is required by WordPress and is only used to load the other main configs. Nothing else should be added to it.
 
-`config/application.php` is the main config file that contains what `wp-config.php` usually would. Base options should be set in there.
+The file to modify for configuration options is `config/application.php`. This file that contains what `wp-config.php` usually would.
 
-For environment specific configuration, use the files under `config/environments`. By default there are `development`, `staging`, and `production` but these can be whatever you require.
+These base configuration options should be production-standard _SAFE SETTINGS_ and will be used in all environments except where specifically overridden.
 
-The environment configs are required **before** the main `application` config so anything in an environment config takes precedence over `application`.
+For environment-specific configuration option **overrides**:
 
-Note: You can't re-define constants in PHP. So if you have a base setting in `application.php` and want to override it in `production.php` for example, you have a few options:
+  * Use or create files under `config/environments`. The `config/application.php` code will require any file in the `config/environments` directory with a filename matching the `WP_ENV` assignment. This will either be in a `.env` file at the top of the Bedrock installation or for a Trellis installation, defined in the `wordpress_sites` settings.
 
-* Remove the base option and be sure to define it in every environment it's needed
-* Only define the constant in `application.php` if it isn't already defined.
\ No newline at end of file
+  * Bedrock installs with `development.php` and `staging.php` files. If you want to add have a `WP_ENV=my-foo-bar` environment, you can add those options in `config/environments/my-foo-bar.php`.
```

Comment:
> Bedrock comes with `development.php` and `staging.php` configs included. If you want to create an additional environment, create it in `config/environments`.

Reply:
Great. Does this seems like too much?

Bedrock comes with development.php and staging.php configs included. If you want to 
create an additional environment, configure it with a matching php file in config/environments.

Reply:
That works 👍